release_images_training.yml

```diff
@@ -1,8 +1,8 @@
 ---
 release_images:
   1:
-    framework: "tensorflow"
-    version: "2.13.0"
+    framework: "pytorch"
+    version: "2.0.1"
```

Comment:
@dkey-amazon Could you please create separate specs for PT 2.0.1 SM DLCs specifically, so that the TF 2.13 release specs can continue to exist for whatever releases need to be made?